### ISSUE TITLE
False error message during Recognition

### ISSUE THREAD

**Issue by maxnth** [OCR4all/OCR4all#41]
When recognizing pages the console output sometimes stops and the following error message is displayed. 

> Error
> 
> The execution of the Recognition process failed.
> Please check for status errors and try to execute again later.

After closing the error message, going to "Process Overview" and then going back to "Recognition" one can observe, that the Recognition process hasn't stopped and is running as expected. The console output continues as intended.

**Comment by maxnth** [comment]
The same behaviour can be seen during _Line Segmentation_.

**Comment by maxnth** [comment]
The bug has now been reported for every step of the workflow. 
Switching to another entry in the workflow menu (e. g. _Line Segmentation_ to _Project Overview_) and then switching back always solves the problem. 
Might be caused by time out during the requests.

**Comment by emanuel-22** [comment]
Hello good, I have a page segmentation error. The following happens to me:
I complete the ground truth of the pages. Then in project Overview what has been done, both the segmentation and the line segmentation are lost. The error occurs in the XML, in its generation. Any solution?

**Comment by maxnth** [comment]
>Then in project Overview what has been done, both the segmentation and the line segmentation are lost.

This definitely shouldn't happen.
Can you let us know which version of OCR4all you used?
Did this happen more than once?

**Comment by emanuel-22** [comment]
Hi, the version of ocr4all is 0.4.0 with the same version of Larex and we mount it to a server.
I comment better:
1) Perform all the Process Flow of all the pages
2) I did the GT Production through Larex of all the pages
3) In Project Overview, I cross-mark the Line Segmentation and GT columns of some pages.
4) I do the process flow only of those specific pages
5) Mark me with a check mark up to the Line Segmentation column
6) I do GT of those specific pages
7) I go back to Project Overview, and I cross-mark the Line Segmentation and the GT of those pages

Another thing, do 2 times the process flow of all the pages and they are specific pages. Another thing I realized is in the generation of XML because I get this error for example if I want to do a training:

File "src / lxml / parser.pxi", line 654, in lxml.etree._raiseParseError
  File "/var/ocr4all/data/Rodrigo/processing/0006.xml", line 1
lxml.etree.XMLSyntaxError: Start tag expected, '<' not found, line 1, column 55

 I hope someone had a similar problem.

Greetings from Argentina!
